aspnetcore/security/cors/sample/Cors/ClientApp/Pages/Index2.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@page
```

Comment:
Is this file checked in by accident? The file name seems suspect and you usually don't share models.

Reply:
Index2 is used imported into the docs. It's uses

`onclick="requestVal('https://<web app>.azurewebsites.net/api/values')" />`

Which is not a valid URL. Index 1 has the code I test - which I cleaned up the URL as you suggest - now that I have delete the Azure resource.

`onclick="requestVal('https://webapi123.azurewebsites.net/api/widget/1')" />`